CONTRIBUTING.md

```diff
@@ -39,6 +41,25 @@ To send us a pull request, please:
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
+## Contributing a new Kubectl Asset
+If you would like to contribute a new Kubectl Asset with a different kubectl version, please follow these steps:
+
+1. Open a new GitHub issue titled `Feature Request: Asset with kubectl vX.Y.Z`. We will track the progress of the new
+Asset in the issue.
+2. If we decide to support the requested version, a maintainer will open a new branch, `kubectl-vY/main`
+(Y is the minor version) and update the issue accordingly.
+3. Fork the repository and fetch the `kubectl-vY/main` branch locally, and modify the source off of that.
+4. Specifically: 
+    - change `README.md` to reflect the new versions of kubectl and helm that the asset will include.
+    - change `KUBECTL_VERSION` and `HELM_VERSION` in `layer/Dockerfile` to reflect the new versions.
+    - change `SPEC_VERSION` in `.projenrc.js` to reflect the new minor version of kubectl.
```

Comment:
Explain how to choose a helm version that is compatible. Link to the relevant docs.